1

```diff
@@ -0,0 +1,220 @@
+<?php
```

Comment:
What about this for example if you scroll down

http://www.sitecrafting.com/blog/php-patterns-part-ii/